src/app/core-ui/main/status/modal/help-modal/console-modal.component.ts

```diff
@@ -117,17 +117,33 @@ export class ConsoleModalComponent implements OnInit, AfterViewChecked {
     }
   }
 
+  manageCommandHistory(code: number) {
+    if (code === 38) {
+      if (this.historyCount > 0) {
+        this.historyCount--;
```

Comment:
is it a counter or an index ?

Reply:
Its a counter

Reply:
is it not the current index of the historic command ? counter is to count stuff, for instance "there is x transactions to zap" while index can go up and down to navigate an array.

Reply:
yes it is managing indexing stuff for an array but is a counter which increases and decreases on the basis of up and down